include/util/static_rtree.hpp

```diff
@@ -65,11 +79,20 @@ class StaticRTree
     using EdgeData = EdgeDataT;
     using CoordinateList = Vector<util::Coordinate>;
 
+    static constexpr auto UNALIGNED_LEAF_PAGE_SIZE =
+        BRANCHING_FACTOR * sizeof(EdgeDataT) + sizeof(Rectangle) + sizeof(std::uint32_t);
```

Comment:
Yes, that was my next thought.  I'm going to remove the `LeafNode` object and just use  `TreeNode` all the way down (with an `is_leaf`) flag.  This will allow for variable size leaf nodes.

My hope is that the performance gains from the better packing and smaller node sizes will outweigh the benefits of the improved disk cache interaction.